### DISCUSSION TITLE
Create professionally designed Websites! blazor-cms.org 🚀with Oqtane 💧

### DISCUSSION THREAD

**Discussion by iJungleboy** [oqtane/oqtane.framework#2495] [Show and tell]
We just published blazor-cms.org 🚀

The goal is to promote Oqtane and provide guidance how to use Oqtane to quickly create...

1. awesome looking websites
2. professionally designed and responsive
3. with Content Management
4. SEO
5. ...and everything :)

![image](https://user-images.githubusercontent.com/5321186/201474721-926ebbf0-7c98-47a4-a2ee-f276d55c3096.png)

It's still WIP, but please check it out and leave some feedback here what you think, areas we should focus on etc.

Also check out our [blog post about blazor-cms.org](https://2sxc.org/en/blog/post/demo-of-oqtane-as-cms-blazor-cms-org)

💓 Oqtane!

**Comment by ADefWebserver** [comment]
For your Get Started, I would also have a Github "Install" button that uses: [Creating a Custom Distribution of Blazor Oqtane Using Site Templates](https://blazorhelpwebsite.com/ViewBlogPost/41) to allow them to create an Oqtane installation with your modules already installed.

**Reply by iJungleboy** [reply]
@ADefWebserver this is awesome, I wasn't aware of this bundling functionality. 💪🏽

We'll definitely have a look at this, because it would make getting started much faster and easier!

@tvatavuk do check this out 🚀

**Comment by iJungleboy** [comment]
To anybody watching: this is also an important SEO with Oqtane experiment, and we're tweaking the details as we go. I'll keep you posted.

**Reply by iJungleboy** [reply]
SEO seems to work ;)!

![image](https://user-images.githubusercontent.com/5321186/202383595-1e493398-79bd-48ba-aa3c-5f8bff4a4878.png)

**Reply by iJungleboy** [reply]
Just an intermediate report: the site has been online for about 2 monhs now, and any search for blazor cms gives this as a top result. We're really happy about this :)

**Reply by iJungleboy** [reply]
Update: Still going strong. We plan to publish a blog next week and then we can do extensive content / SEO testing.

**Comment by Raceeend** [comment]
![image](https://user-images.githubusercontent.com/669420/201943165-2d3a44f7-d6a2-4cc4-9f53-e2de5c52e26f.png)

The screenshot shows me the doubts I have in using Oqtane for the development of our websites. (to be clear the error is caused by our corporate firewall blocking websockets)
I like Oqtane, I like the fantastic framework - more precise modular application framework. It's exactly what I was looking for. 
But.... it doesn't work for all our customers. Just like us, it appears that some of our customers also block websockets. And this means that the website will not work. A product that you know will not always work is not acceptable.

The question here is, how do we deal with this, is this solvable?

**Reply by iJungleboy** [reply]
This is an interesting point and I don't know the answer. 
To be honest this hasn't been on my radar yet, and I'm not sure if this is a common issue. 
But I do see that even though WebSockets is here to stay, it's probably hard to scan at the firewall. 
Let's open an issue and see what we find out...

**Reply by iJungleboy** [reply]
I created an issue #2500

**Reply by iJungleboy** [reply]
I'm not sure if this is a relevant issue, but I'll try to look more into in in a few weeks.

**Comment by iJungleboy** [comment]
There is a Meetup to discuss CMS functionality on Oqtane on Friday 2nd December

Join us :) https://www.facebook.com/groups/dnnconnect/permalink/2891792887633999/

https://www.meetup.com/2sxc-for-dnn-and-oqtane/?fbclid=IwAR0lb3P2necOxaGf4ieDKtjhOAitidC9LEO5qN15VPXLS3ynFO1H4hkps4Y

**Reply by iJungleboy** [reply]
FYI: Meeting was a big success. We'll continue with more of these.

**Comment by iJungleboy** [comment]
Just released the blog https://blazor-cms.org/blog

It's using the standard [2sxc Blog App](https://2sxc.org/apps/app/blog-v5-hybrid-for-dnn-and-oqtane) and we're curious how it will perform in terms of SEO.

**Comment by iJungleboy** [comment]
I also just published the [Oqtane tutorials](https://blazor-cms.org/oqtane-tutorials)

This should give people about 200 pages of Razor tutorials and best practices ;)

**Reply by thabaum** [reply]
@iJungleboy strange thing happening at that page when you try to use the link to go back to blazor-cms home from that tutorials page users can not navigate to the homepage.

**Reply by iJungleboy** [reply]
thx! fixed. 
It's an issue with linking between sites in Oqtane, which doesn't quite do what I think it should, so I had to add `?reload` for anybody fighting with the same issue.

**Comment by iJungleboy** [comment]
Update: Still going strong, with latest Oqtane 5 and 2sxc 17 - Google Indexing working as it should 🔥
